mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/utbetaling/model/DelutbetalingDto.kt

```diff
@@ -1,89 +1,29 @@
+    val utbetalingId: UUID,
+    val status: DelutbetalingStatus,
+    val periode: Periode,
+    val belop: Int,
+    val frigjorTilsagn: Boolean,
+    val lopenummer: Int,
+    val fakturanummer: String,
+)
 
-    @Serializable
-    @SerialName("DELUTBETALING_TIL_GODKJENNING")
-    data class DelutbetalingTilGodkjenning(
-        @Serializable(with = UUIDSerializer::class)
-        override val id: UUID,
-        @Serializable(with = UUIDSerializer::class)
-        override val tilsagnId: UUID,
-        @Serializable(with = UUIDSerializer::class)
-        override val utbetalingId: UUID,
-        override val belop: Int,
-        override val frigjorTilsagn: Boolean,
-        override val periode: Periode,
-        override val lopenummer: Int,
-        override val fakturanummer: String,
-        override val opprettelse: Totrinnskontroll,
-    ) : DelutbetalingDto()
-
-    @Serializable
-    @SerialName("DELUTBETALING_OVERFORT_TIL_UTBETALING")
-    data class DelutbetalingOverfortTilUtbetaling(
-        @Serializable(with = UUIDSerializer::class)
-        override val id: UUID,
-        @Serializable(with = UUIDSerializer::class)
-        override val tilsagnId: UUID,
-        @Serializable(with = UUIDSerializer::class)
-        override val utbetalingId: UUID,
-        override val belop: Int,
-        override val frigjorTilsagn: Boolean,
-        override val periode: Periode,
-        override val lopenummer: Int,
-        override val fakturanummer: String,
-        override val opprettelse: Totrinnskontroll,
-    ) : DelutbetalingDto()
-
-    @Serializable
-    @SerialName("DELUTBETALING_UTBETALT")
-    data class DelutbetalingUtbetalt(
-        @Serializable(with = UUIDSerializer::class)
-        override val id: UUID,
-        @Serializable(with = UUIDSerializer::class)
-        override val tilsagnId: UUID,
-        @Serializable(with = UUIDSerializer::class)
-        override val utbetalingId: UUID,
-        override val belop: Int,
-        override val frigjorTilsagn: Boolean,
-        override val periode: Periode,
-        override val lopenummer: Int,
-        override val fakturanummer: String,
-        override val opprettelse: Totrinnskontroll,
-    ) : DelutbetalingDto()
-
-    @Serializable
-    @SerialName("DELUTBETALING_AVVIST")
-    data class DelutbetalingAvvist(
-        @Serializable(with = UUIDSerializer::class)
-        override val id: UUID,
-        @Serializable(with = UUIDSerializer::class)
-        override val tilsagnId: UUID,
-        @Serializable(with = UUIDSerializer::class)
-        override val utbetalingId: UUID,
-        override val belop: Int,
-        override val frigjorTilsagn: Boolean,
-        override val periode: Periode,
-        override val lopenummer: Int,
-        override val fakturanummer: String,
-        override val opprettelse: Totrinnskontroll,
-    ) : DelutbetalingDto()
+enum class DelutbetalingStatus {
+    TIL_GODKJENNING,
+    GODKJENT,
+    RETURNERT,
+    UTBETALT,
```

Comment:
Skal vi bake status fra oebs-faktura inn i denne? 

Her har vi foreløpig UTBETALT, burde vi da også ha f.eks. `SENDT_TIL_UTBETALING` og `UTBETALING_FEILET` for å indikere disse mulighetene også?